CONTRIBUTORS.md

```diff
@@ -99,6 +100,7 @@ Created by running `./build-support/bin/contributors.sh`.
 + Kushal Dave
 + Kyle Derr
 + Larry Hosken
++ LarryFinn
```

Comment:
It looks like Larry might need a `.mailmap` entry. His self-selected full name in slack (x2) is 'Lawrence Finn'.